### ISSUE TITLE
Validation Result Issues

### ISSUE THREAD

**Issue by skylineo** [fudan-zvg/DeMo#4]
Thank you for your previous response! I ran your code, and it took about 10 hours on two RTX 3090s. However, I’m a bit confused about the val_new metric during validation. Could you clarify what it represents?
Additionally, I've noticed that the validation results are slightly worse than those reported in the paper. The only change I made was increasing the Batch_Size in the config from 16 to 64 while keeping everything else the same. Could this be related to the drop in performance? The validation result is shown below.
![image](https://github.com/user-attachments/assets/42ab032a-f8b6-4420-ac21-11d760c32aad)

**Comment by Zbozhou** [comment]
Thanks. 'val_' represents the trajectory prediction results from the mode query, while 'val_new' indicates the final trajectory prediction results. Regarding performance, we also observed some fluctuations with different batch sizes and varying numbers of GPUs during the experiments, though the fluctuations were not significant.

**Comment by skylineo** [comment]
Oh, I see! Thank you so much for your timely response!

**Comment by skylineo** [comment]
I'm sorry to bother you again. I've been testing and submitting using the weights trained earlier, but I noticed that my results are significantly worse than yours. Could this be due to factors like the number of GPUs? What strategies might help improve the performance? Here’s the configuration I used:
defaults:
  - datamodule: av2  # 'av2' for 'DeMo', 'av2_stream' for 'DeMo+RealMotion'
  - model: model_forecast  # 'model_forecast' for 'DeMo', 'stream_model_forecast' for 'DeMo+RealMotion'
# eg: For 'DeMo'
# defaults:
#   - datamodule: av2
#   - model: model_forecast

hydra:
  run:
    dir: outputs/${output}/${now:%Y%m%d-%H%M%S}

seed: 2024
monitor: val_minFDE6
save_top_k: 10

data_root:
checkpoint:
pretrained_weights:
output: change_${datamodule.name}_${datamodule.phase}_${model.name}_${model.phase}

# trainer
num_workers: 8
gpus: [1,2]  # when test, only use 1 gpu; when train/val, use k gpus
sync_bn: true
batch_size: 64
epochs: 60
warmup_epochs: 10

# optimizer
lr: 0.003  
weight_decay: 1e-2  
gradient_clip_val: 5
gradient_clip_algorithm: norm

limit_train_batches:
limit_val_batches:
limit_test_batches:
log_model: all
test: false  # false for val, true for test

Comparison:
![image](https://github.com/user-attachments/assets/bb7d88fd-1cbf-4bce-a2d5-b5acaa0d71b9)
![image](https://github.com/user-attachments/assets/634ebafa-2ce0-4984-b584-512cef0ed3c6)

**Comment by Zbozhou** [comment]
As we specified in main and appendix, "RealMotion" is used as the baseline of "DeMo" in test, and this baseline will be removed in other times. If you need to submit to leaderboard, you need to use the 'DeMo'+'RealMotion' model.

**Comment by skylineo** [comment]
Very thanks for your answer, helps a lot!
